epub34/overview/index.html

```diff
@@ -675,34 +693,40 @@ <h3>OEB, OCF and EPUB 2: 1999—2010</h3>
 			</section>
 			<section id="epub30">
 				<h3>EPUB 3.0: 2010</h3>
-
-				<p>Work on a major revision of the EPUB specifications began in 2010, with the goal of aligning EPUB
-					more closely with HTML, and in the process bringing new, native multimedia features, sophisticated
+				<p>
+					Work on a major revision of the EPUB specifications began in 2010, with the goal of aligning EPUB
+					more closely with HTML. As part of that alignment, the reference to HTML was changed: while EPUB 2.0 
+					referred to XHTML 1.1 [[xhtml11]], EPUB 3.0 relies on the XML serialization of HTML [[html]] (referred to
+					as "HTML5" at the time). This resulted in bringing new, native multimedia features, sophisticated
 					CSS layout rendering and font embedding, MathML support, scripted interactivity, vertical writing
-					and other enhanced global language support, and improved accessibility. A new specification for EPUB
-					Media Overlays was also introduced, allowing for text and audio synchronization in EPUB
-					publications. To better align the specification names with the standard, the Open Package Format
+					and other enhanced global language support, and improved accessibility. 
+				</p>
+
+				<p>
+					A new specification for EPUB Media Overlays was also introduced, allowing for text and audio synchronization 
+					in EPUB publications. To better align the specification names with the standard, the Open Package Format
 					specification was renamed EPUB publications, and the Open Publication Format specification was
 					renamed EPUB Content Documents. The EPUB 3.0 specifications were approved in October 2011.
 					See [[epubpublications-30]] [[epubcontentdocs-30]] [[ocf-30]] [[epubmediaoverlays-30]]
-					[[epubchanges-30]].</p>
+					[[epubchanges-30]].
+				</p>
 			</section>
 
 			<section id="epub301">
 				<h3>EPUB 3.0.1: 2014</h3>
 
 				<p>The EPUB 3.0.1 revision was undertaken in 2013-14. Although introducing mostly minor fixes and
-					updates, it did see the integration of Fixed Layout Documents, which give publishers greater control
-					over presentation when a reflowable EPUB is not suitable for the content.
+					updates, it did see the integration of Fixed Layout Documents, which gave publishers greater control
+					over presentation when a reflowable EPUB was not suitable for the content.
```

Comment:
The change to past tense here makes it sound like fixed layouts no longer give this control. Their benefit isn't in the past, only the change that added them.

It's also a mouthful. Maybe shorten this to:

```suggestion
					updates, it did see the integration of fixed layout documents to provide greater control
					over the presentation of EPUB publications.
```